MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -103,7 +103,7 @@
     @param  error       The error that occurred creating the application object, if any (optional)
  */
 - (nullable instancetype)initWithClientId:(nonnull NSString *)clientId
```

Comment:
what's the reason to deprecate this one? It seems simple enough and most of developers are using this one right now. I think Android was planning to keep this one around as well.

Reply:
done